docs/config_reference.rst

```diff
@@ -1006,6 +1006,20 @@ Common scheduler options
    In such cases, you may set this parameter to ``true`` to avoid this.
 
 
+.. js:attribute:: .schedulers[].resubmit_on_qos_errors
+
+   :required: No
+   :default: ``[]``
+
+   This option is relevant to the Slurm backends only.
+
+   When a job is submitted certain errors can be ignored and the framework will try to submit again the job after some seconds.
```

Comment:
Not `certain` errors -> QOS errors. And we should give an example in parenthesis.

Reply:
Or we change the option name to `resubmit_on_errors`, which I think corresponds better to what we actually do.

Reply:
I think it's better to rename the option to `resubmit_on_errors` and keep it more general.